modules/nf-core/ribowaltz/templates/ribowaltz.r

```diff
@@ -0,0 +1,479 @@
+#!/usr/bin/env Rscript
+
+# Calculation of optimal P-site offsets, diagnostic analysis and visual inspection of ribosome profiling data
+# Author: Ira Iosub
+# Example usage: Rscript ribowaltz.r --bam SRX11780887.Aligned.toTranscriptome.out.bam --gtf Homo_sapiens.GRCh38.111_chr20.gtf --fasta Homo_sapiens.GRCh38.dna.chromosome.20.fa.gz --length_range "26:34" --periodicity_threshold 50 --exclude_start 42 --exclude_stop 27
```

Comment:
This is never going to be called this way as an nf-core template, so this should be removed.

Reply:
deleted. Referencing commit 2c16f8e.